users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -55,16 +57,23 @@ def __init__(
             - 'minimum' (float): The minimum preemphasis factor.
             - 'maximum' (float): The maximum preemphasis factor.
             Example: {"prob": 0.9, "minimum": 0.9, "maximum": 1.0}
+        :param non_linearity: A dictionary containing parameters for the non-linearity filter.
+            - 'prob' (float): The probability of applying the non-linearity filter.
+            - 'alpha' (float): The alpha value for the non-linearity filter.
```

Comment:
Like for other perturbations, please define alpha via a minimum and maximum and sample it randomly from that range. To have a fixed alpha, you can then still set both minimum and maximum to that value.